gf-guide/metrics.md

```diff
@@ -31,11 +31,21 @@ Please note that the first metrics guidelines referred to in this guide are Lati
 
 The following rules apply to all new font families, and should be enforced to upgraded font families when possible.
 
-**1. Vertical metrics must not be calculated by the font editor automatically.**
+#### 1. Vertical metrics must not be calculated by the font editor automatically
 
-GF doesn't want this because they all do it differently.
+This is an overall best practice in design. When defining line spacing, it's important to strike a balance between legibility and avoiding overlapping letters. At GF, we have established a specific system that prioritizes optimal performance for various situations, while still leaving room for future update improvements like adding more languages without causing regressions. 
```

Comment:
This is an issue all over the docs, but using there term GF might be too much jargon for people not involved in communicating about Google Fonts regularly. I made an issue here about this: https://github.com/googlefonts/googlefonts.github.io/issues/93 

If you read the GitHub Docs for example, they use the full name GitHub regularly instead of saying GH, but I'm almost certain GH is used in internal conversations.
 
https://docs.github.com/en

Reply:
I would change `...for future update improvements like adding...` to `...for future improvements, like adding...`

Reply:
Done.